lib/signaling/v2/networkqualitysignaling.js

```diff
@@ -131,6 +144,22 @@ class NetworkQualitySignaling extends EventEmitter {
     }
     if (level !== null && this.level !== level) {
       this._level = level;
+      updated = true;
+    }
+
+    let remoteLevels = null;
+    const remotes = message ? message.remotes : null;
+    if (remotes) {
+      remoteLevels = remotes.reduce((map, obj) => {
```

Comment:
@syerrapragada Nitpick: Instead of `map` in the `reduce()` callback, can the name be more descriptive like `levels`?